### DISCUSSION TITLE
Show tree with another onClick event

### DISCUSSION THREAD

**Discussion by alvaroponguta** [dowjones/react-dropdown-tree-select#490] [Q&A]
Hello everybody! Im trying to show the tree without taking into account the onClick event of the input, and trigger it from another onClick event of another button im rendering beside this component. Is that possible?

**Comment by mrchief** [comment]
Depends on your use case but you may want to look at [showdropdown](https://github.com/dowjones/react-dropdown-tree-select#showdropdown) options. When you render the component `showdropdown={'initial'}` will show the dropdown in opened state.
